### ISSUE TITLE
sandworm-audit run terminates with TypeError: process.stdout.moveCursor is not a function

### ISSUE THREAD

**Issue by OleksiyRudenko** [sandworm-hq/sandworm-audit#103]
**Sandworm version**
v1.36.0

**Describe the bug**
Error report in terminal (git bash under Windows 10)
```shell
$ sandworm-audit
Sandworm 🪱
Security and License Compliance Audit
- Building dependency graph...

//
// 💡 Mark issues as resolved with Sandworm
//    sandworm resolve ISSUE-ID
//    https://docs.sandworm.dev/audit/resolving-issues
//
C:\Users\xxx\AppData\Roaming\nvm\v16.19.1\node_modules\@sandworm\audit\src\cli\progress.js:63
              process.stdout.moveCursor(0, -6);
                             ^

TypeError: process.stdout.moveCursor is not a function
    at Timeout._onTimeout (C:\Users\xxx\AppData\Roaming\nvm\v16.19.1\node_modules\@sandworm\audit\src\cli\progress.js:63:30)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)

```

**Manifest files**
Whenever possible, please provide your `package.json` manifest file content (or at least a list of all the dependencies within), as well as your lockfile.

**To Reproduce**
In terminal (git bash under Windows 10):
1. Install sandworm with `npm i -g @sandworm/audit`
2. Navigate to arbitrary project that qualifies requirements
3. Run `sandworm-audit`

**Expected behavior**
Task completed without termination at building dependency graph phase.

**System (please complete the following information):**
- Node Version 16.19.1
- Package Manager **npm**
- Package Manager Version 8.19.3
- OS: Windows 10

Node and npm are under [nvm](https://github.com/coreybutler/nvm-windows) management.

**Additional context**
This was the first run right after fresh installation of the package.

**Comment by gabidobo** [comment]
Thank you for reporting this, @OleksiyRudenko!
Looking into it now.

**Comment by gabidobo** [comment]
Fixed in v1.36.2.
@OleksiyRudenko if you get a chance, please let me know if the fix works for you! Thanks.
